dephell_pythons/_finder.py

```diff
@@ -108,14 +109,11 @@ def get_implementation(path: Path) -> Optional[str]:
                 return implementation
         return None
 
-    def is_python(self, path: Path) -> bool:
-        # https://stackoverflow.com/a/377028/8704691
-        try:
-            if not path.is_file():
-                return False
-        except PermissionError:
-            return False
-        if not os.access(str(path), os.X_OK):
+    def is_python(self, path: Path, force: bool = False) -> bool:
```

Comment:
I find `force` arg and condition down below where it used a bit confusing

At first I interpreted that `force=True` indicates, that we will force check that file is exists before running any logic.

I guess to get rid of such a duality we should have more explicit name for it, like `ensure_exists` (any suggestion on new are welcomed). 

What do you think?
```suggestion
    def is_python(self, path: Path, ensure_exists: bool = True) -> bool:
```